inference-engine/src/transformations/include/transformations/utils/utils.hpp

```diff
@@ -131,6 +135,24 @@ Output<Node> eltwise_fold(const Output<Node> & input0, const Output<Node> & inpu
 
 TRANSFORMATIONS_API std::vector<Input<Node>> get_node_target_inputs(const std::shared_ptr<Node>& node);
 
+
+template<class SrcOpClass, class DstOpClass>
+bool replace_operation(ngraph::pattern::Matcher& m,
```

Comment:
Why do we need to have this function?
In the majority of transformations we still use `replace_node`

Reply:
>Why do we need to have this function?

to reduce code duplication. So that we do not have to call in stereotyped transformations every time 
`m.get_match_root()`
...
`set_friendly_name()`
`copy_runtime_info()`
`replace_node()`

Do you have objections to put this to the common utils?

Reply:
communicated with @GlebKazantaev on email. Removed this function from `ie utils`.